src/components/Personalization/createOnClickHandler.js

```diff
@@ -35,10 +35,10 @@ export default ({
         const xdm = { eventType: INTERACT };
         const scope = decisionsMeta[0].scope;
 
-        if (scope !== PAGE_WIDE_SCOPE) {
+        if (isNonEmptyString(scope) && scope !== PAGE_WIDE_SCOPE) {
           xdm.web = {
             webPageDetails: {
-              viewName: scope
+              viewName: scope.toLowerCase()
```

Comment:
`scope` is stored lowercase, no need for this here.

Reply:
Fixed in #1056